scripts/install-mender.sh

```diff
@@ -57,13 +57,18 @@ Running the Mender installation script.
 }
 
 usage() {
-    echo "usage: install-mender.sh [-h] [-c channel] [component...]"
+    echo "usage: install-mender.sh [options] [component...] [-- [options-for-mender-setup] ]"
+    echo ""
+    echo "options: [-h, help] [-c channel] [--demo] [--commercial]"
     echo "  -h, --help          print this help"
     echo "  -c CHANNEL          channel: stable(default)|experimental"
     echo "  --demo              use defaults appropriate for demo"
```

Comment:
is there a chance to reuse this setting in the setup args? it does look a bit erroneous to have `--demo` twice in [here](https://github.com/mendersoftware/gui/pull/1978/files#diff-0b474816e57448323f2fe953ebd8a228af18faa82b2650929f25d52f6a0c3235R113)
if we would align on setting `--demo` for hosted & on-prem enterprise I think it would be as easy as appending `--demo` to the `$MENDER_SETUP_ARGS`

Reply:
Yeah, I know and I thought about it, but unfortunately `--demo` means different things in the script and in `mender setup` - for the Enterprise setup we want "script demo" but "production setup". See [related comment](https://github.com/mendersoftware/gui/pull/1978#discussion_r706039700).